guides/developer/extending/integrate-your-CI/index.md

```diff
@@ -0,0 +1,126 @@
+---
+layout: single
+title:  "Integrate your CI"
+sidebar:
+  nav: guides
+---
+
+{% include toc %}
+
+At Netflix, we built CI integrations to enable features like CI in Spinnaker, fetch artifact metadata and more. You can see the full list of CI features [here](/guides/user/managed-delivery/CI-features).
+
+This guide will explain how to add an integration to your own CI provider, so you'll be able to enjoy the new features we provide.
+
+## CI view in Spinnaker
+
+We recently added the option to see CI details in Spinnaker, in a new `Builds` tab.
+It looks like this:
+{%
+  include
+  figure
+  image_path="./ci-view.png"
+%}
+
+This is a new tab that can be accessed from the main Spinnaker UI.
+The UI code (in deck) is calling `gate` service to fetch information, and `gate` is just calling `igor`.
+
+If you wish to use it, this is what you'll need to do:
+0. Make sure you are using [gate](https://github.com/spinnaker/gate). Here's the [CiController](https://github.com/spinnaker/gate/blob/master/gate-web/src/main/groovy/com/netflix/spinnaker/gate/controllers/CiController.java) the UI is calling (no need to do any actions here).
```

Comment:
Is this relevant at all? I don't think you can have a Spinnaker installation without gate, can you?

Reply:
I'm not sure, but is there harm in mentioning it?

Reply:
Any time you include information that's not relevant to the discussion, there's a potential for confusion, that's why I asked. I'd suggest removing.